src/commands/flex/plugins/start.ts

```diff
@@ -1,15 +1,76 @@
+import { join } from 'path';
+
+import { flags } from '@oclif/command';
+
 import { createDescription } from '../../../utils/general';
-import FlexPlugin from '../../../sub-commands/flex-plugin';
+import FlexPlugin, { ConfigData, SecureStorage } from '../../../sub-commands/flex-plugin';
+import { filesExist } from '../../../utils/fs';
+
 /**
  * Starts the dev-server for building and iterating on a flex-plugin
  */
 export default class FlexPluginsStart extends FlexPlugin {
-  static description = createDescription('Starts a dev-server to build the Flex plugin locally', true);
+  static description = createDescription('Starts a dev-server to build the Flex plugin locally', false);
+
+  static flags = {
+    ...FlexPlugin.flags,
+    name: flags.string({
+      multiple: true,
+      char: 'n',
+    }),
+    'include-remote': flags.boolean(),
+  };
+
+  constructor(argv: string[], config: ConfigData, secureStorage: SecureStorage) {
+    super(argv, config, secureStorage, {});
+
+    if (this._flags['include-remote'] || this._flags.name) {
+      this.opts.runInDirectory = false;
+    }
+  }
 
   /**
    * @override
    */
   async doRun() {
-    await this.runScript('start');
+    const names = this._flags.name;
+    const includeRemote = this._flags['include-remote'];
+    const flexArgs: string[] = [];
+    const pluginArgs: string[][] = [];
+
+    if (names) {
```

Comment:
Not needed - the parser will ensure this is always an array.

Reply:
I get a parsing error without the if statement

Reply:
Not sure what you mean

Reply:
without `if (this._flags.name)` I get the error:
`Unexpected error occurred
TypeError: this._flags.name is not iterable`

Reply:
Weird okay